app/src/main/java/org/mozilla/rocket/theme/ThemeManager.java

```diff
@@ -0,0 +1,136 @@
+package org.mozilla.rocket.theme;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.preference.PreferenceManager;
+
+import org.mozilla.focus.R;
+
+import java.util.HashSet;
+
+public class ThemeManager {
+
+    private static final int ONBOARDING_VERSION = 1;
+
+    private static final String PREF_KEY_STRING_CURRENT_THEME = "pref_key_string_current_theme";
```

Comment:
I forgot what was the conclusion. Should we move these to the xml?

Reply:
I think this preference is more like a status config, not user modifiable settings. It's not used as a key of PreferenceItem. We probably should add a button in settings to allow user reset all the onboarding to learn what feature/on-boarading has been missing. Then, we can inject this key to that onboarding manager. Would that be better? @mTwTm

Reply:
Per what I've heard from @cnevinc he seems to believe these prefs should stay there too. At least that's what he told me when I was implementing unread screenshots. I think showing users the old features is a good idea but controlling it with PREF_KEY_STRING_CURRENT_THEME sounds a bit cumbersome to me. We'll need to handle more pairs of from-versions and to-versions. Maybe we should simply add them into settings so users can read anything anytime? Well, this may not work well with home onboarding and screenshot onboarding which are not shown in Cardviews, though.

Reply:
My preference on this topic is I don't think it really matters where that key resides. It's more like a coding style to me so we can change at any time and I'm just wondering if we have that rule yet.